src/plugins/home/server/services/sample_data/routes/uninstall.ts

```diff
@@ -61,6 +61,12 @@ export function createUninstallRoute(
         }
       }
 
+      const includedHiddenTypes = sampleDataset.savedObjects
+        .map((object) => object.type)
+        .filter((supportedType) => typeRegistry.isHidden(supportedType));
```

Comment:
Allows uninstalling all sample types, regardless if it is not hidden or not importable/exportable